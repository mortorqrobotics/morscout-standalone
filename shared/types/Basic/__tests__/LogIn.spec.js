import {
  login,
  loggedin,
  logout,
  loggedout,
  invalidCredentials
} from "../LogIn";

test("Login stays the same", () => {
  expect(login).toMatchSnapshot();
});

test("Loggedin stays the same", () => {
  expect(loggedin).toMatchSnapshot();
});

test("Logout stays the same", () => {
  expect(logout).toMatchSnapshot();
});

test("Loggedout stays the same", () => {
  expect(loggedout).toMatchSnapshot();
});

test("Invalid Credentials stay the same", () => {
  expect(invalidCredentials).toMatchSnapshot();
});
