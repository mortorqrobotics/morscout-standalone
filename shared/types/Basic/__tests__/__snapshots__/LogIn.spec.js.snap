// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Invalid Credentials stay the same 1`] = `"error/INVALID_LOGIN_CREDENTIALS"`;

exports[`Loggedin stays the same 1`] = `"LOGGEDIN"`;

exports[`Loggedout stays the same 1`] = `"LOGGEDOUT"`;

exports[`Login stays the same 1`] = `"server/LOGIN"`;

exports[`Logout stays the same 1`] = `"server/LOGOUT"`;
